src/java.base/share/classes/java/util/List.java

```diff
@@ -1190,4 +1195,68 @@ static <E> List<E> of(E... elements) {
     static <E> List<E> copyOf(Collection<? extends E> coll) {
         return ImmutableCollections.listCopy(coll);
     }
+
+    /**
+     * {@return a new lazily computed list with the provided {@code size}}
```

Comment:
A minor comment here is that  "of the given size" might be a bit clearer than "with the provided size".